centreon/www/class/centreon-clapi/centreonLDAP.class.php

```diff
@@ -333,51 +315,52 @@ public function setparam($parameters = array())
         }
         $arId = $this->getLdapId($params[0]);
         if (is_null($arId)) {
-            throw new CentreonClapiException(self::OBJECT_NOT_FOUND . ":" . $params[0]);
+            throw new CentreonClapiException(self::OBJECT_NOT_FOUND . ':' . $params[0]);
         }
-        if (in_array(strtolower($params[1]), array('name', 'description', 'enable'))) {
-            if (strtolower($params[1]) == 'name') {
-                if (!$this->isUnique($params[2], $arId)) {
+        if (in_array(mb_strtolower($params[1]), ['name', 'description', 'enable'], true)) {
+            if (mb_strtolower($params[1]) === 'name') {
+                if (! $this->isUnique($params[2], $arId)) {
                     throw new CentreonClapiException(self::NAMEALREADYINUSE . ' (' . $params[2] . ')');
                 }
             }
             $this->db->query(
```

Comment:
Please use a prepare + execute to avoid this legacy call to a modified query signature from PDO 🙏